routes/defRoute.js

```diff
@@ -9,56 +9,76 @@ const router = express.Router();
 
 router.get('/', async (req, res) => {
   try {
-    const defibrillators = await Defibrillator.find();
+    const defibrillators = await Defibrillator.find().select(
+      'address title location'
+    );
     return res.status(200).send(defibrillators);
   } catch (e) {
     errorHandler(res, e);
   }
 });
 
-router.post('/', /*passport.authenticate('jwt', { session: false }),*/ async (req, res) => {
-  try {
-    const defibrillator = await Defibrillator.create(
-      req.body,
-    );
-    return res.status(201).send({
-      error: false,
-      defibrillator,
-    });
-  } catch (e) {
-    errorHandler(res, e);
+router.post(
+  '/',
+  /*passport.authenticate('jwt', { session: false }),*/ async (
+    req,
+    res
+  ) => {
+    try {
+      const defibrillator = await Defibrillator.create(
+        req.body
+      );
+      return res.status(201).send({
+        error: false,
+        defibrillator
+      });
+    } catch (e) {
+      errorHandler(res, e);
+    }
   }
-});
+);
 
-router.put('/:id', /*passport.authenticate('jwt', { session: false }),*/ async (req, res) => {
-  try {
-    const { id } = req.params;
-    const defibrillator = await Defibrillator.findByIdAndUpdate(
-      id,
-      req.body,
-    );
-    return res.status(200).send({
```

Comment:
For version of Mongoose 4.0, You need explicit set the option new to true, like
 const defibrillator = await Defibrillator.findByIdAndUpdate(
          id,
          req.body,
          {new: true}
 );
Link to official docs: https://mongoosejs.com/docs/api.html#query_Query-findOneAndUpdate

Reply:
Thanks. Added